uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences-my-leave-calendar.js

```diff
@@ -1,9 +1,91 @@
+/* globals jQuery */
+
 var page = require('./page');
 
 module.exports = (function () {
   return page.extend({
     /**
-     * Wait for the page to be ready by looking at 
+     * Clears all selected months from the calendar "Selected Months" field.
+     *
+     * @returns {Object} - returns a reference to itself.
+     */
+    clearAllSelectedMonths: function () {
+      this.casper.evaluate(function () {
+        jQuery('.chr_leave-calendar__day-selector .close.ui-select-match-close')
```

Comment:
Why do we need jQuery here, why cant we use `casper.click`

Reply:
Because `casper.click` only clicks one element and to clear all months you need to click on the close icon for each one of the months that are selected.

Reply:
By default a single month is selected, why do we need to clear multiple months?

Reply:
You are right, I'll clear only the currently selected month